src/api_gateway/utils/otp_structure.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+type ReqSendCode struct {
+	Phone string `json:"phone"`
```

Comment:
Make the login endpoint receiving field name universal for both email and phoneNumber (maybe loginValue).